src/celeritas/Types.hh

```diff
@@ -122,6 +122,15 @@ enum class StepPoint
     size_
 };
 
+//---------------------------------------------------------------------------//
+//! Ordering / sorting of tracks on GPU
+enum class TrackOrder
+{
+    unsorted,
+    shuffled,
+    size_
+};
```

Comment:
When I originally wrote this comment I was going to suggest changing `arbitrary` to `unsorted` so good job beating me to that 😄 I like `shuffled` better than `random` too.